job_executor/model/datastore_version.py

```diff
@@ -163,6 +163,12 @@ def set_draft_release_status(self, dataset_name: str, new_status: str):
                 f'Status already set to {new_status}'
             )
         dataset_update.set_release_status(new_status)
-        self.release_time = self._get_current_epoch_seconds()
+        self._set_release_time(self._get_current_epoch_seconds())
         self._calculate_update_type()
         self._write_to_file()
+
+    def _set_release_time(self, epoch_seconds):
+        self.release_time = epoch_seconds
```

Comment:
I think it might as well call the self._get_current_epoch_seconds() here to avoid passing the same parameter every time 👍🏻

Reply:
Yes, good catch